features/dd-sdk-android-rum/src/test/kotlin/com/datadog/android/rum/internal/ndk/DatadogNdkCrashEventHandlerTest.kt

```diff
@@ -147,6 +147,7 @@ internal class DatadogNdkCrashEventHandlerTest {
             verify(mockRumWriter, times(2)).write(eq(mockEventBatchWriter), capture())
 
             ErrorEventAssert.assertThat(firstValue as ErrorEvent)
+                .hasApplicationId(fakeViewEvent.application.id)
```

Comment:
looks like redundant line

Reply:
Hrm good call. I'll write a test for it and make sure it defaults back to ndk.

Reply:
Minor change to a default so I'm asking for a re-review.

Reply:
I mean here the same assertion `hasApplicationId(fakeViewEvent.application.id)` comes twice

Reply:
.... oh... weird! I think github had an issue. I was responding to the comment about the throw 😅